bg.js

```diff
@@ -0,0 +1,17 @@
+//Listens for messages from content scripts
+chrome.runtime.onMessage.addListener(function(request, sender) {
+    if(request.action == "checkAudioPlaying") {
+        var tabName = request.tabName;
+
+        //Find tab object from the playing tab (by its title)
+        chrome.tabs.query({title: tabName}, function(tabs) {
```

Comment:
I think using `chrome.tabs.getCurrent` and `chrome.tabs.get` with the ID is better to avoid duplication or others problems

Reply:
The problem, as I've tried to describe it in the issue is that content scripts don't allow the use of `chrome.tabs` at all, hence `chrome.tabs.getCurrent` won't work there. This was my first approach, and it always returned `undefined` for `chrome.tabs`. This also won't work in bg.js itself, as this background script runs independent from any tabs.

https://stackoverflow.com/a/35523438 This answer describes the same thing and gives a neat little snippet to work around this. I'll try to implement that approach, it will get rid of possible duplication issues as well.